wormhole-connect/src/utils/sdkv2.ts

```diff
@@ -192,6 +195,15 @@ const parseTokenBridgeReceipt = async (
     txData.tokenKey = tokenV1.key;
     txData.receivedTokenKey = tokenV1.key;
     txData.receiveAmount = txData.amount;
+    txData.receiveNativeAmount = Number(
```

Comment:
Here I added parsing for `receiveNativeAmount` and `relayerFee` to `parseTokenBridgeReceipt`. This is needed when resuming a transaction that takes user to the Redeem view (see `views/TxSearch.tsx`).
I believe we need to add a similar parsing to `parseCCTPReceipt` and `parseNttReceipt` as well. @artursapek if that's the case, can you point me to sample transaction configs for those two routes so I can debug, rev-eng and add those parsing logic as well?

Reply:
@artursapek I did the same to figure out where to get those amounts in the attestation object. Which network/token configurations will show me CCTP and NTT routes?

Reply:
I've also added an item to track this work for prod release:
https://www.notion.so/wormholelabs/UI-Feature-Add-receiveNativeAmount-and-relayerFee-to-CCTP-and-NTT-parsers-in-utils-sdkv2-ts-b9c0018830da4160b962535e2e76f20a?pvs=4